core/src/apresolve.rs

```diff
@@ -59,14 +57,6 @@ async fn try_apresolve(
     Ok(ap)
 }
 
-pub async fn apresolve(proxy: Option<&Url>, ap_port: Option<u16>) -> String {
-    try_apresolve(proxy, ap_port).await.unwrap_or_else(|e| {
-        warn!("Failed to resolve Access Point: {}", e);
-        warn!("Using fallback \"{}\"", AP_FALLBACK);
-        AP_FALLBACK.into()
-    })
-}
```

Comment:
As long as the fallback ap works sometimes, why not keeping this fallback instead of failing immediately if apresolve would be down?

Reply:
We shortly discussed this in https://github.com/librespot-org/librespot/discussions/734#discussioncomment-767901.

Sure we could make it infallible again in the way you proposed. But when `apresolve.spotify.com` never seems to fail and is what's actually used by official Spotify clients, we might as well remove the cruft.

Reply:
Are you staying that a secondary access point, that could be used when the first one is [down](https://community.spotify.com/t5/Spotify-for-Developers/Expired-certificate-on-apresolve-spotify-com/td-p/5007278) is cruft? Do not agree.

Reply:
I thought to take pre-emptive action when `ap.spotify.com` is on the way out. But it's based on an assumption that is only founded on the fact that @devgianlu doesn't have it in `librespot-java` (which I take as a reference for the new API). Do you think it's a _secondary_ access point ("feature") or a _deprecated_ access point ("cruft")?

I see it's working again today:
```
 $ date
Tue  1 Jun 10:52:24 CEST 2021
$ openssl s_client -connect ap.spotify.com:443
CONNECTED(00000003)
```

If we don't have signs that it's being decommissioned then sure, let's change it into a true fallback without feature-gating `apresolve` and we can always change later.

Reply:
> Do you think it's a secondary access point ("feature") or a deprecated access point ("cruft")?

I don't see why it matters, this is a tiny amount of uncomplicated code. It doesn't seem worth a breaking change to me. None of us here know what anything is there for and what is deprecated, this is all a best guess. But it is there and it works just fine as a backup. It's likely this static endpoint is less complicated than the geo-aware one so it could be argued it's got a non-zero chance of still working in a backup situation. The good news for us is that history shows Spotify never turn anything off, even officially deprecated stuff - they have too many legacy installs to support.

Reply:
Fair enough. I'll change later today.